### DISCUSSION TITLE
Problems reading image

### DISCUSSION THREAD

**Discussion by redstonerti** [ImageMagick/ImageMagick#3725] [Help]
I can't get image.read to work no matter what i try, and it's the simplest part of the api.
I always get the same error while debugging:
![image](https://user-images.githubusercontent.com/48157961/119241324-44c4cc00-bb5e-11eb-89c3-19c2c581eaf9.png)
And when building the project, i get 56 warnings:
![image](https://user-images.githubusercontent.com/48157961/119241640-a128eb00-bb60-11eb-9046-03002090b7cc.png)
I'm on windows, visual studio code with cl.exe as my configuration.
Copying all the dlls to the same folder as the executable didn't help. 
Adding this to tasks didn't help:
```
"-LC:\\Program Files\\ImageMagick-7.0.11-Q16-HDRI\\lib",
"-lCORE_RL_Magick++_",
"-lCORE_RL_MagickCore_",
"-lCORE_RL_MagickWand_",
```
This is my main
```c++
using namespace Magick;
using namespace std;
namespace fs = std::filesystem;
int main(int argc, char **argv)
{
    cout << *argv << endl;
    InitializeMagick(*argv);
    char *appdata = getenv("APPDATA");
    char *desktop = getenv("USERPROFILE");
    string new_appdata = appdata;
    string new_desktop = desktop;
    new_desktop += "\\Desktop";
    cout << "Desktop: " << new_desktop << endl;
    string screenshot_string = new_appdata + "/.minecraft/screenshots";
    for (const auto &entry : fs::directory_iterator(screenshot_string))
    {
        string path = entry.path().string();
        std::replace(path.begin(), path.end(), '/', '\\');
        path = ToLowerCase(path.substr(0, 1)) + path.substr(1, path.length() - 1);
        cout << "File path: " << path << endl;
        Image image;
        image.read(path);
    }
    cout << "Press any key to end the process..." << endl;
    char a = _getch();
    return 0;
}
```

**Comment by dlemstra** [comment]
Is it possible to create a small visual studio project that can be used to reproduce this, and share it on GitHub?
